pkg/controller/constraint/constraint_controller.go

```diff
@@ -280,12 +282,6 @@ func (r *ReconcileConstraint) Reconcile(ctx context.Context, request reconcile.R
 		}
 	}()
 
-	ct := &v1beta1.ConstraintTemplate{}
-	err = r.reader.Get(ctx, types.NamespacedName{Name: strings.ToLower(instance.GetKind())}, ct)
-	if err != nil {
-		return reconcile.Result{}, err
-	}
-
```

Comment:
this whole block should be restored. we need to continue to reconcile until the CRD for this constraint is there.